### ISSUE TITLE
Updating upstream dependency

### ISSUE THREAD

**Issue by spiroski** [samjarrett/FractalBundle#10]
Can you release a new version which uses league/fractal 0.17.0.
They introduced a new resource type called Primitive which is quite useful.

**Comment by samjarrett** [comment]
Hey @spiroski, the composer requirement is `^0.15`, which would include 0.17.0 (includes up to 1.0.0). Unless something is broken with newer versions, you should only need to `composer update` to get it.

**Comment by samjarrett** [comment]
(If there's actually something broken, let me know and I'll reopen/fix - thanks!)

**Comment by spiroski** [comment]
I'm not sure what I'm doing wrong, but if I do `composer update league/fractal` i'm getting this:

```
composer update league/fractal --ignore-platform-reqs
Loading composer repositories with package information
Updating dependencies (including require-dev)
Nothing to install or update
Writing lock file
Generating autoload files
```

And if I try to require `league/fractal` I'm getting this error:

```
composer require league/fractal --ignore-platform-reqs
Using version ^0.17.0 for league/fractal
./composer.json has been updated
Loading composer repositories with package information
Updating dependencies (including require-dev)
Your requirements could not be resolved to an installable set of packages.

  Problem 1
    - samj/fractal-bundle 1.0.0 requires league/fractal ^0.15 -> satisfiable by league/fractal[0.15.0] but these conflict with your requirements or minimum-stability.
    - samj/fractal-bundle 1.0.0 requires league/fractal ^0.15 -> satisfiable by league/fractal[0.15.0] but these conflict with your requirements or minimum-stability.
    - samj/fractal-bundle 1.0.0 requires league/fractal ^0.15 -> satisfiable by league/fractal[0.15.0] but these conflict with your requirements or minimum-stability.
    - Installation request for samj/fractal-bundle ^1.0.0 -> satisfiable by samj/fractal-bundle[1.0.0].

```

I'd prefer not to change my minimum-stability.

**Comment by samjarrett** [comment]
OK, according to the composer [version constraints documentation](https://getcomposer.org/doc/articles/versions.md#caret-version-range-):

> For pre-1.0 versions it also acts with safety in mind and treats ^0.3 as >=0.3.0 <0.4.0

I'll update and release a new version to support 0.17.

**Comment by spiroski** [comment]
Thank you.

I think if you want to replicate the caret functionality for pre 1.0 releases, you could use something like `0.*`.

**Comment by samjarrett** [comment]
Hey @spiroski, I've pushed an update to a branch - `2.x`, since I don't quite know how the primitives work, would you mind testing & letting me know if there's any issues? If all good, I'll release it as 2.0.0.

**Comment by samjarrett** [comment]
Have released as version [2.0.0](https://github.com/samjarrett/FractalBundle/releases/tag/2.0.0).
